### ISSUE TITLE
Tap a station add a bubble with address on top of it

### ISSUE THREAD

**Issue by EtienneM** [ThePublicBikeGang/EasyBike#112]
J'ai fait jouer ma copine avec l'app et le truc qui la perturbe c'est que quand tu appuies sur une station il ne se passe rien d'autre que l'action bar qui arrive. Elle s'attend à avoir des infos dans une petit bulle au dessus. Ça pourrait être :
* Adresse
* Nombre de vélos disponible / Nombre de place total
* ...

**Comment by MarcMagnin** [comment]
yes adresse pourquoi pas, en revanche la quantité de vélo est déjà affichées dessus :)

**Comment by MarcMagnin** [comment]
j'ai ajouté la bulle pour l'adresse
